tests/configuration/default_local_config.yml

```diff
@@ -0,0 +1,9 @@
+MQ:
```

Comment:
Consider adding this section to `default_core_conf.yml`. In the future, we might consider moving all communication params to a separate file (Websocket, GUI websocket, MQ, etc.)